js/ui/popover.js

```diff
@@ -91,13 +92,7 @@ var POPOVER_CLASS = "dx-popover",
             action({ event: e, target: $(e.currentTarget) });
         };
 
-        if(target.jquery || target.nodeType || typeUtils.isWindow(target)) {
-            that["_" + name + "EventHandler"] = undefined;
-            eventsEngine.on(target, eventName, handler);
-        } else {
-            that["_" + name + "EventHandler"] = handler;
-            eventsEngine.on(domAdapter.getDocument(), eventName, target, handler);
```

Comment:
You missed the case when the target element is set as a selector. If this item is not in the DOM at the time of code execution, the Tooltip will not be shown

Reply:
Thanks, it's fixed now

Reply:
Thanks, it's fixed now